project_event/models/project_project.py

```diff
@@ -136,7 +136,7 @@ def get_message(self, action):
         return {
             'body': self.get_message_body(action) + message,
             'channel_ids': [(6, 0, [self.env.ref
-                            ('project.mail_channel_project_task').id])],
+                            ('project.mail_channel_project_task_event').id])],
```

Comment:
you call it mail_channel_project_task2 or mail_channel_project_task_event

Reply:
mail_channel_project_task_event